reactor-core/src/test/java/reactor/core/publisher/FluxRetryPredicateTest.java

```diff
@@ -36,7 +37,7 @@ public void sourceNull() {
 	@Test(expected = NullPointerException.class)
 	public void predicateNull() {
 		Flux.never()
-		    .retry(null);
+		    .retry((Predicate<? super Throwable>) null);
```

Comment:
I wonder if it may cause some source compatibility issues for the users too

Reply:
users are not supposed to pass in `null` from the API documentation, this is just validating that if they misbehave badly the operator application is rejected